src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -623,41 +623,53 @@ namespace shogun
 		}
 
 		/**
-		 * Compute the eigenvalues and eigenvectors of a symmetric matrix.
+		 * Compute the top-k eigenvalues and eigenvectors of a symmetric matrix.
```

Comment:
Eigen computes all the eigenvalues and dsyevr should too but for the latter I'm not 100% so I'll double check.